src/Features/Lsif/Generator/Program.cs

```diff
@@ -26,17 +28,18 @@ public static Task Main(string[] args)
                 new Option("--solution", "input solution file") { Argument = new Argument<FileInfo>().ExistingOnly() },
                 new Option("--project", "input project file") { Argument = new Argument<FileInfo>().ExistingOnly() },
                 new Option("--compiler-invocation", "path to a .json file that contains the information for a csc/vbc invocation") { Argument = new Argument<FileInfo>().ExistingOnly() },
+                new Option("--binlog", "path to a MSBuild binlog that csc/vbc invocations will be extracted from") { Argument = new Argument<FileInfo>().ExistingOnly() },
```

Comment:
@KirillOsenkov Is binlog one word or two? :smile:

Reply:
I use it as a single word "binlog" :)